kahuna/public/js/components/gr-image-usage/gr-image-usage.js

```diff
@@ -58,12 +58,19 @@ module.directive('grImageUsage', [function() {
     };
 }]);
 
-module.controller('grImageUsageListCtrl', [function () {
+module.controller('grImageUsageListCtrl', [
+    'imageUsagesService',
+    function (imageUsagesService) {
     const ctrl = this;
 
     ctrl.formatTimestamp = (timestamp) => {
         return moment(timestamp).fromNow();
     };
+
+    ctrl.isRecent = (timestamp) => {
+        var nowtime = new Date();
+        return moment(timestamp).isBetween(moment(nowtime).subtract(imageUsagesService.recentTime,'days'),moment(nowtime));
```

Comment:
Can we use isAfter like below, I think it's easier to understand what's going on